app/styles/_components.less

```diff
@@ -281,6 +281,19 @@ code.probe-command {
 }
 
 .pod-donut {
+  .c3-defocused {
+    // Removing hover opacity change which caused the "c3-Running" state to look very similar to "c3-Not-Ready" state
+    // https://github.com/openshift/origin/issues/9253
+    opacity: 1 !important;
+  }
+  .c3-tooltip-container {
+    // centered above donut
+    left: 50% !important;
+    top: -10px !important;
+    transform: translate(-50%, -50%);
+    // prevent text wrap
+    white-space: nowrap;
+  }
```

Comment:
Can you remove the following lines with this change (and the trailing comma on the line above)?

https://github.com/spadgett/origin-web-console/blob/339a32eff93ba3a1298717fe2e9f82f55ea0d884/app/scripts/directives/podDonut.js#L84-L87